duties.md

```diff
@@ -0,0 +1,24 @@
+# Team duties
+
+This document is helpful to know what areas are covered by each team member.
```

Comment:
```suggestion
This document details which areas are covered by each team member.
Besides overall transparency, this might come in handy in case of vacations or sickness.
```